src/ScheduleService.Models.Db/DbTask.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Db;
+
+public class DbTask
+{
+  public const string TableName = "Tasks";
+
+  public Guid Id { get; set; }
+
+  [MaxLength(20)]
+  public string Name { get; set; }
+
+  [MaxLength(300)]
+  public string Description { get; set; }
+
+  public int Type { get; set; }
+
+  public Guid GroupId { get; set; }
+  public Guid WorkspaceId { get; set; }
+
+  public DbGroup Group { get; set; }
+  public DbWorkspace Workspace { get; set; }
+  public IList<DbCategory> Categories { get; set; } = new List<DbCategory>();
```

Comment:
split and order like this:

public const string TableName = "Tasks";

public Guid Id { get; set; }
public Guid GroupId { get; set; }
public Guid WorkspaceId { get; set; }
public string Name { get; set; }
public string Description { get; set; }
public int Type { get; set; }

public DbGroup Group { get; set; }
public DbWorkspace Workspace { get; set; }
public IList<DbCategory> Categories { get; set; } = new List<DbCategory>();

Reply:
I have questions
Will restrictions on properties be set in validation then? Don't you need to add them in the attributes? And then configure the restrictions in the database in the migration file?

Reply:
Such restrictions are useful in requests coming from the front. There they will actually check the data. Here it’s too late.